src/applications/simple-forms/mock-alternate-header-21-0845/components/CustomFormApp.jsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
```

Comment:
overall strategy for this PR was to use custom components inside this mock form directory rather than global/shared files, until this is proved to be a good solution.